addons/sale_crm/wizard/crm_opportunity_to_quotation.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Opportunity2Quotation(models.TransientModel):
+
+    _name = 'crm.quotation.partner'
+    _description = 'Create new or use existing Customer on new Quotation'
+    _inherit = 'crm.partner.binding'
+
+    def _default_lead_id(self):
```

Comment:
Do something like this https://github.com/odoo/odoo/blob/12.0/addons/sale_timesheet/wizard/project_create_invoice.py#L13 ;)